src/schema/purchaseOrder/types.ts

```diff
@@ -132,5 +134,33 @@ export const PurchaseOrderLoadable = builder.loadableObject(PurchaseOrderRef, {
         return userTickets.map((ut) => selectUserTicketsSchema.parse(ut));
       },
     }),
+    userTicketAddons: t.loadableList({
+      type: UserTicketAddonRef,
```

Comment:
CREO que esto podría ser:

```
type: [UserTicketAddonRef]
resolve: async (root, s, { DB }) => {
  const ticketAddons = await DB.query.ticketAddonsSchema.findMany({
    where: (ta, { eq, and }) =>
       and(eq(ta.purchaseOrderId, root.purchaseOrder.id)),
  });
  
  return ticketAddons.map((ta) => selectTicketAddonSchema.parse(ta));
},

Reply:
Es importante que los data loaders devuelvan los items en el mismo orden que los ids que vienen por parametros.

Devolverlos de esta manera no aseguraria eso ya que el findMany no asegura el orden.

Anteriormente me pasó que tuve que arreglar estos problemas por que estaba obteniendo información que no era la correcta en el #258 

Fuente https://pothos-graphql.dev/docs/plugins/dataloader
`It is VERY IMPORTANT to return values from load in an order that exactly matches the order of the requested IDs. The order is used to map results to their IDs, and if the results are returned in a different order, your GraphQL requests will end up with the wrong data. Correctly sorting results returned from a database or other data source can be tricky, so there this plugin has a sort option (described below) to simplify the sorting process. For more details on how the load function works, see the [dataloader docs](https://github.com/graphql/dataloader#batch-function).`